src/core/appdir.cpp

```diff
@@ -767,6 +769,10 @@ namespace linuxdeploy {
                 return d->deployFile(from, to, true);
             }
 
+            void AppDir::symlinkFile(const bf::path& target, const bf::path& symlink, const bool useRelativePath) {
+                d->symlinkFile(target, symlink, useRelativePath);
```

Comment:
This function returns `bool` to allow for error checks. That value should be available to the caller. Otherwise, they cannot know whether the function actually created a symlink or not, and hence can't write a good error check.